packages/cli/templates/webcomponents/igc-ts/grid/default/files/src/app/__path__/__filePrefix__.ts

```diff
@@ -45,9 +45,8 @@ export default class $(ClassName) extends HTMLElement {
       :host {
         height: 100%;
         margin: 0px;
-        padding-left: 275px;
         padding-right: 20px;
-        width: calc(100% - 275px);
+        width: calc(100% - 600px);
```

Comment:
Woah that looks weird. I guess it can be looked at separately though

Reply:
Removing the styles in styles.css and app.ts broke the grid views. That's why I updated the width here, No other views were affected. However, you are right this does not look nice. I will look into the grid issue further,